src/EnergyPlus/Plant/Component.cc

```diff
@@ -121,5 +121,9 @@ namespace DataPlant {
         }
     }
 
+    CompData &CompData::getPlantComponent(EnergyPlusData &state, PlantLocation const &plantLoc) {
+        return DataPlant::CompData::getPlantComponent(state, plantLoc);
```

Comment:
🤦🏽 Function definition got changed in global find and replace. Fixing.

Reply:
I was going to say something ... but I figured the infinite recursion (or g++) would say it first.